cmake/AddResources.cmake

```diff
@@ -45,14 +45,20 @@ function(add_resources target)
 			# of the directory
 			install(DIRECTORY "assets/${dir}"
 				DESTINATION "${INSTALL_APPDATADIR}"
-				COMPONENT "${target}"
 			)
 		endif()
 	endforeach()
 
 	foreach(file IN LISTS ARG_FILES)
 		target_sources(${target} PRIVATE "assets/${file}")
 		set_property(TARGET ${target} APPEND PROPERTY RESOURCE "assets/${file}")
+		get_filename_component(file_dir "${file}" DIRECTORY)
+		if(NOT "${file_dir}" STREQUAL "")
+			set(file_dir "/$file_dir")
```

Comment:
I am blind and did not notice this is not a variable until now, it is replacing `file_dir` with literally `/$file_dir` (should be `/${file_dir}`). This if-branch shouldn’t be needed at all though.

Reply:
Yeah evidently it doesn't end up getting called, so I'll just delete it.